services/ui-src/src/utils/state/useUser.ts

```diff
@@ -0,0 +1,25 @@
+import { create } from "zustand";
+import { devtools, persist } from "zustand/middleware";
+// types
+import { MfpUserState, MFPUser } from "types";
+
+export const useUser = create(
```

Comment:
General zustand question: Should state initialization still be a use____ style hook? Or is this something that only needs to be initialized one time?